scalafix-sbt/src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -50,6 +56,19 @@ object ScalafixPlugin extends AutoPlugin {
     cliWrapperMainClass := "scalafix.cli.Cli$",
     scalafixEnabled := true,
     scalafixVerbose := false,
+    scalafixBuild := Def.inputTaskDyn {
```

Comment:
Would it be worth it to check the `scalacOptions` to make sure that `sbthost` is present and fail early if not? What happens if someone runs `scalafixBuild` without the compiler plugin at the moment?